maven/bnd-maven-plugin/src/main/java/aQute/bnd/maven/plugin/BndMavenPlugin.java

```diff
@@ -222,7 +229,7 @@ private static void expandJar(Jar jar, File dir) throws Exception {
 					continue;
 			}
 
-			IO.copy(resource.openInputStream(), outFile);
+			IO.copy(resource.openInputStream(), buildContext.newFileOutputStream(outFile));
```

Comment:
@cschneider Did you plan to address my comment above about closing the output streams?